addons/medical_treatment/CfgWeapons.hpp

```diff
@@ -1,32 +1,31 @@
 
 class CfgWeapons {
     class ItemCore;
+    class ACE_ItemCore;
     class CBA_MiscItem_ItemInfo;
     class InventoryFirstAidKitItem_Base_F;
     class MedikitItem;
 
-    // ITEMS
     class FirstAidKit: ItemCore {
         type = 0;
-        ace_arsenal_hide = 1;
+        EGVAR(arsenal,hide) = 1;
         class ItemInfo: InventoryFirstAidKitItem_Base_F {
             mass = 4;
         };
     };
     class Medikit: ItemCore {
         type = 0;
-        ace_arsenal_hide = 1;
+        EGVAR(arsenal,hide) = 1;
```

Comment:
I think we should port over the removal of these entries from #7017

My reasoning is that there are a number of reasons people might still want to add vanilla items to units that will be only replaced if ACE is present (e.g. it's possible to make missions addon compatible but not dependent). Additionally it will be consistent with any other addons that add to the replacement config because they won't be hiding their own items.